src/SvelteTable.svelte

```diff
@@ -317,17 +316,25 @@
             handleClickRow(e, row);
           }}
           class={asStringArray([
-            classNameRow,
+            typeof classNameRow === "string" ? classNameRow : null,
+            typeof classNameRow === "function" ? classNameRow(row, n) : null,
+            ,
```

Comment:
Why empty row?

Reply:
I missed this comment somehow. Removed the empty row in latest release.